### PR TITLE
[#32028] Create panel layout home page

### PR DESCRIPTION
**Pull request by HierenFeb**
Create panel layout for home page.
Login and go https://agov_pr513.agov.com.au/admin/structure/pages/nojs/operation/page-front_page/handlers/page_front_page_panel_context/layout

### CODE REVIEW

**Comment by HierenFeb** [comment]
test this please

**Comment by HierenFeb** [comment]
test this please

**Comment by HierenFeb** [comment]
test this please

**Comment by joannguyen** [comment]
layout homepage
The layout is enough to implement homepage

**Comment by joannguyen** [comment]
Hey Feb, please create 3 columns in bottom content (3 box items)

**Comment by joannguyen** [comment]
test this please

**Comment by joannguyen** [comment]
home layout
Great!
@JohnAlbin Can you check and merge

File: themes/agov/agov_whitlam/panels-layouts/home/agov-layouts-home.tpl.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+<div class="gov-home-layout layout-3col clearfix" <?php if (!empty($css_id)) : print "id=\"$css_id\""; endif; ?>>
+
+  <?php if (!empty($content['header'])) : ?>
+    <div class="layout-3col__full">
+      <?php print $content['header'];?>
+    </div>
+  <?php endif; ?>
+
+  <?php if(!empty($content['main']) || !empty($content['right-sidebar'])) : ?>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/panels-layouts/home/agov-layouts-home.tpl.php]: Why the extra PHP if wrapper?

File: themes/agov/agov_whitlam/panels-layouts/home/agov-layouts-home.tpl.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * @license GPL v2 http://www.fsf.org/licensing/licenses/gpl.html
+ *
+ * Available variables
+ * -------------------
+ * $content array of panels.
+ */
+
+?>
+
+<div class="gov-home-layout layout-3col clearfix" <?php if (!empty($css_id)) : print "id=\"$css_id\""; endif; ?>>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/panels-layouts/home/agov-layouts-home.tpl.php]: What is "gov-home-layout" class needed for? Go ahead and remove it.

**Pull request merged**